### ISSUE TITLE
Separate video recording

### ISSUE THREAD

**Issue by pospelov14203** [StepicOrg/Stepik-Studio#60]
It would be nice to be able to make only camera recording or screencast recording.
Also display accessibility of camera/tablet - if some of this is inaccessible "start recording" interface element of corresponding device should be locked.
